src/frontend/src/api/ResourceCollection.ts

```diff
@@ -84,6 +84,18 @@ export default class ResourceCollection {
         );
     }
 
+    delegateRoles(projectId: string, contractId: string, search?: string) {
+       const url = combineUrls(this.contract(projectId, contractId), 'delegated-roles');
+       if(!search) {
+           return url
+       }
+       return url + search
```

Comment:
unsure of the endpoint, but shouldn't  it be wrapped in combineUrls(url, search) to ensure trailing slash in url?

Reply:
The search is being used as a url query string. Changing name from "search" to "query" to make it more understandable